exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -151,41 +143,37 @@ public bool WasCorrectlyAnswered()
                 if (isGettingOutOfPenaltyBox)
                 {
                     Console.WriteLine("Answer was correct!!!!");
-                    purses[currentPlayer]++;
-                    Console.WriteLine(players[currentPlayer]
+                    playersPurses[currentPlayer]++;
```

Comment:
Reading this line alone doesn't make it clear what is happening. Maybe extract a method: MoveToNextPlayerPurse()